OrbitService/LinuxTracingHandler.cpp

```diff
@@ -143,6 +144,22 @@ void LinuxTracingHandler::OnTracepointEvent(orbit_grpc_protos::TracepointEvent t
   capture_event_buffer_->AddEvent(std::move(event));
 }
 
+void LinuxTracingHandler::OnMapsUpdate(uint64_t timestamp, const std::string& maps) {
+  auto result_or_error = utils::ParseMaps(maps);
```

Comment:
This is related to the other comment. This class does buffering and interning. This processing doesn't really belong here, it seems like it's done here just to use `ServiceUtils.h`. Is there a reason why this processing shouldn't be done already in `OrbitLinuxTracing`?

Reply:
we need to extract additional information (load_bias and build_id), this is not currently available by parsing done in LinuxOrbitTracing, also note that OrbitLinuxTracing produces libunwindstack related structures which we need to convert and complete. It is possible to move ParseMaps to OrbitLinuxTracing but it would also introduce some additional dependencies.

Reply:
> OrbitLinuxTracing produces libunwindstack related structures which we need to convert and complete

But those are only used internally by `OrbitLinuxTracing` and don't leave it, or am I missing what you meant?

> It is possible to move ParseMaps to OrbitLinuxTracing but it would also introduce some additional dependencies.

Are there particular dependencies other than `ElfFile`? Considering that `OrbitLinuxTracing` depends on `libunwindstack`, I wouldn't be against depending also on `ElfFile` or the like.

In general `LinuxTracingHandler` seems a bit of a limbo to put this processing.

Reply:
see above.